tensorboard/webapp/widgets/linked_time_fob/linked_time_types.ts

```diff
@@ -20,6 +20,16 @@ export interface LinkedTime {
   end: {step: number} | null;
 }
 
+/**
+ * The linked time fob controller can be used inside both horizontal and
+ * vertical axis. This enum is a utility used to inform to the controller of
```

Comment:
nit: the description is a bit confusing to me. It sound like "linked time fob controller" is used in "axis" but I think you mean the other way?
And for type definition, I think it is okay to just say what it is first, and add more about how it might be used later if it's not sufficient to just say "what it is"

for example,
The direction of axis used to control the fob movements.

Reply:
Changed.

The Controller does actually live "inside" the axis. But that is mostly academic.